hail_search/queries/base.py

```diff
@@ -393,6 +399,14 @@ def _merge_project_hts(project_hts, n_partitions, include_all_globals=False):
     def _filter_entries_table(self, ht, sample_data, inheritance_filter=None, quality_filter=None, **kwargs):
         ht = self._prefilter_entries_table(ht, **kwargs)
 
+        # Temporarily reset sample_data
```

Comment:
it might be helpful to include an indicator of how temporary it is (i.e "until full blended eS/GS support is added" or "until lookup table supports sample type")